dap-mode.el

```diff
@@ -1240,7 +1241,14 @@ DEBUG-SESSION is the active debug session."
              #'equal))
 
 (defun dap--set-exception-breakpoints (debug-session callback)
-  (-let [(&dap-session 'initialize-result 'launch-args (&plist :type)) debug-session]
+  ;; XXX: for whatever reason, make byte-compile emits warnings when
```

Comment:
This seems weird, the production code has been proven working, perhaps we need to tune/require/shuffle something for this to work?